e2e/jest.test-setup.ts

```diff
@@ -20,130 +20,6 @@ beforeEach(async () => {
   page.setDefaultNavigationTimeout(60000); // Puppeteer default timeout is 30 seconds.
   await page.setRequestInterception(true);
 
-  const getPageTitle = async () => {
```

Comment:
Moving function to outside of `beforeEach` block because they don't belong in `beforeEach`